components/Layout/Images/LazyImage.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <img :data-src="src" :alt="alt" class="lazy-hidden" @error="onError" />
```

Comment:
You're emiting an event here, but the `onError` method is never called. so this seems unnecessary

Reply:
The ``img`` element itself is capable of emitting error events, that's why. Although we add the ``src`` attribute afterwards, I'm not sure if that makes the element emit errors when they occur.

@Camc314 @MrTimscampi Any idea about this?